deploy/runtime/startup.lk

```diff
@@ -115,16 +115,19 @@ function setup_battery_pages_behind_the_meter()
 			   'Battery REopt Optimization',
 			   'Battery Current and Capacity',
 			   'Battery Configuration',
-			   'Battery Dispatch Common',
-			   'Battery Dispatch Behind the Meter',
-			   'Battery Dispatch Manual',
                {'name'='Battery Lifetime', 'caption'='Battery Lifetime', 'collapsible'=true, 'collapsible_var'='battery_lifetime_shown'},
 			   {'name'='Battery Replacements', 'caption'='Battery Replacements', 'collapsible'=true, 'collapsible_var'='battery_replacements_shown'},
 			   {'name'='Battery Voltage', 'caption'='Battery Voltage', 'collapsible'=true, 'collapsible_var'='battery_voltage_shown'},
 			   {'name'='Battery Ancillary Losses', 'caption'='Battery Losses', 'collapsible'=true, 'collapsible_var'='battery_losses_shown'},
 			   {'name'='Battery Thermal', 'caption'='Battery Thermal', 'collapsible'=true, 'collapsible_var'='battery_thermal_shown'},
 			  ]],
 			  { 'sidebar'='Battery Storage', 'help'='battery_storage_btm' } );
```

Comment:
@cpaulgilman @brtietz Can I suggest calling this "Battery Cell and System" to align with the battery's SSC group names?